torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -2370,6 +2396,20 @@ RegisterOperators reg2({
     CREATE_LIST_OPS("bool", c10::List<bool>),
     CREATE_LIST_OPS("Tensor", c10::List<at::Tensor>),
     CREATE_LIST_OPS("t", c10::List<IValue>),
+
+    // `listContains<T>` is not implemented for non-primitive types
+    Operator(
+        "aten::__contains__(int[] l, int item) -> bool",
```

Comment:
why not `bool` type? although it's kinda rare...

Reply:
The cast to a `c10::List<bool>` is broken right now for some reason, we can look into it in a follow up #26036